### ISSUE TITLE
Problems according to IBM Accessibility Checker

### ISSUE THREAD

**Issue by MarkFalconbridge** [gpbl/react-day-picker#1073]
Using the IBM Accessibility Checker (https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp) raises two issues with the day picker component.

1. Interactive component does not have a programmatically associated name - This relates to the month picker element with the role of "grid".

2. The descendent <div> element with "row" role has no focusable child element - This relates to the weekdays picker.

As it stands I can't fix the first of those using the current api and while I _might_ be able to fix the second by replacing the weekday component the default behaviour should have a focusable element (if "row" is the correct role to use for the parent element).

Repro steps, install the IBM Accessibility Checker and run it against the Simple calendar example (http://react-day-picker.js.org/examples/basic) and inspect the results.

I'm aware that there is a V8 of react-day-picker in development so I understand if these issues aren't fixed for version 7 but they should certainly be considered while developing V8.

**Comment by gpbl** [comment]
Closing as [we've released v8](https://github.com/gpbl/react-day-picker/discussions/1412): this issue belongs to a version which is not longer maintained, and it is likely fixed. In case, please open a new issue. Thanks!
